tools/doc/builddoc.sh

```diff
@@ -17,12 +17,5 @@ export PYTHONPATH=$3/$4:$PYTHONPATH
 # Remove old docs
 rm -rf $3/docs/
 
-# Inference doc
-$1/python -m sphinx -j1 -v -T -b html -d $3/docs/inference/_doctrees/html $2/docs/python/inference $3/docs/inference/html
-$1/python -u $2/tools/doc/rename_folders.py $3/docs/inference/html
-# (cd $3/docs/inference/html && zip -r $3/docs/python_inference_doc.zip .)
-
-# Training doc
-$1/python -m sphinx -j1 -v -T -b html -d $3/docs/training/_doctrees/html $2/docs/python/training $3/docs/training/html
-$1/python -u $2/tools/doc/rename_folders.py $3/docs/training/html
-# (cd $3/docs/training/html && zip -r $3/docs/python_training_doc.zip .)
+$1/python -m sphinx -j1 -v -T -b html -d $3/docs/_doctrees/html $2/docs/python/ $3/docs/html
```

Comment:
-j1 means using one one process to generate the documentation. It should work without it.